tests/artifactorygroup_test.go

```diff
@@ -0,0 +1,78 @@
+package tests
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+
+	"github.com/jfrog/jfrog-client-go/artifactory/services"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGroups(t *testing.T) {
+	t.Run("create", testCreateGroup)
+	t.Run("update", testUpdateGroup)
+	t.Run("delete", testDeleteGroup)
+}
+
+func testCreateGroup(t *testing.T) {
+	groupParams := getTestGroupParams(false)
+	err := testGroupService.CreateGroup(groupParams)
+	defer testGroupService.DeleteGroup(groupParams.GroupDetails.Name)
```

Comment:
The ```DeleteGroup``` API returns an error. Let's not ignore it and asset it, even if we're inside the defer clause.
This applies to other places in this file as well.